packages/aggregate-api/src/offer/arrange.js

```diff
@@ -14,14 +14,14 @@ export const provide = (context) =>
  * @returns {Promise<API.UcantoInterface.Result<API.OfferArrangeSuccess, API.OfferArrangeFailure>>}
  */
 export const claim = async ({ capability }, { arrangedOfferStore }) => {
-  const commitmentProof = capability.nb.commitmentProof
+  const pieceLink = capability.nb.pieceLink
```

Comment:
Can we not suffix with types? `piece: Link` is ok? Why is the size not being passed in this instance?

Reply:
The size is only needed on the aggregate offer for SP validation. Here is when there was already a resolution and we just want to create the receipt with final result. For searching we do not need the size (and I don't even know if by the state we get from the Oracle we would have access to it easily).

`piece` is Object with link and size everywhere else. That is the reasoning to name it this way here as we decided in https://github.com/web3-storage/specs/pull/62

Reply:
I got to say `pieceLink` also bothered me, I was not going to say anything but since @alanshaw did how about we rename `piece` object into something else ? I think few cases we considered `pieceInfo` ?

Naming is so hard 😭 and it's made harder by the fact that piece CID does not fully identify the piece without the size....